idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/hetero_reactions.py

```diff
@@ -116,12 +116,16 @@ def build(self):
                                             ("R1", "Sol", "Fe3O4"): 8,
                                             ("R1", "Sol", "Al2O3"): 0}
 
-        # Standard Heat of Reaction - kJ/mol_rxn
-        dh_rxn_dict = {"R1": 136.5843}
+        # Standard Heat of Reaction - J/mol_rxn
+        dh_rxn_dict = {"R1": 136.5843E3}
```

Comment:
A couple of minor comments:

1. Do we really need to declare this separately from its `Param` declaration below?
2. Moe for information that an actual change, but you can add units here too for clarity (i.e. `136.6*pyunits.J/pyunits.mol`). Pyomo will then automatically convert this value to those of the parameter you are assigning them to if required. This is more for use at the flowsheet level, but still applies here.

Reply:
We don't need to define it this way, but it is consistent with how the Parameters are defined in the gas and solid property packages and is flexible for adding more reaction values if needed.